source/agent/analytics/videoGstPipeline/VideoGstAnalyzer.cc

```diff
@@ -14,7 +14,6 @@ namespace mcu {
 DEFINE_LOGGER(VideoGstAnalyzer, "mcu.VideoGstAnalyzer");
 
 GMainLoop* VideoGstAnalyzer::loop = g_main_loop_new(NULL,FALSE);
```

Comment:
You initialize the static variable loop here, then why you do it again at line 134?